scripts/libs/utils.js

```diff
@@ -79,6 +79,33 @@ function getTestnetChains(chains = []) {
     }));
```

Comment:
is this temporary fix needed anymore? we expose the gas service address in the config

Reply:
It's not a fix just a refactoring the same path was already defined as a variable `path`